testsuite/MDAnalysisTests/analysis/test_hole.py

```diff
@@ -74,18 +75,21 @@ def test_HOLE(self, H):
 
         p = profiles_values[0]
 
-        assert len(p) == 425, "wrong number of points in HOLE profile"
+        assert_equal(len(p), 425, err_msg="wrong number of points in HOLE profile")
```

Comment:
Why revert back to `assert_equal`? Didn't we agree that with pytest we should use raw asserts expect when comparing arrays?

Reply:
I thought we said "use `assert_equal()` but not `assert_array_equal()`" – I don't think that bare `assert` can properly compare arrays.

Reply:
... but yeah, in this case this should be `assert`.

Reply:
(I think I used it for making clear where the error message goes... but simple `assert` should also work there.)

I am also replacing `assert_array_*` --> `assert_*`.